tests/test-helper.js

```diff
@@ -1,6 +1,6 @@
-import Application from '../app';
+import Application from '@ember/application';
 import QUnit from 'qunit';
-import config from '../config/environment';
+import config from 'open-event-frontend/config/environment';
```

Comment:
Are you sure this is correct way?

Reply:
@snitin315 , can you please pull these changes to test. it's working as far i tested for everyone's concern if you once please check it.

Reply:
> Are you sure this is correct way?

I think it is, other files of project used this type of import.

Reply:
I'm not sure , but I think somewhere I read that. In ember.js the `project-name/` refers to the  `app` folder by default. Please check if the statement is correct .

Reply:
@snitin315 app.js uses same kind of import - 

```
import Application from '@ember/application';
import Resolver from './resolver';
import loadInitializers from 'ember-load-initializers';
import config from 'open-event-frontend/config/environment';

const App = Application.extend({
  modulePrefix    : config.modulePrefix,
  podModulePrefix : config.podModulePrefix,
  Resolver
});

loadInitializers(App, config.modulePrefix);

export default App;

```

Reply:
[Check This](https://stackoverflow.com/questions/34395590/emberjs-how-to-import-files-using-root-path)

![Screenshot from 2020-02-15 17-03-24](https://user-images.githubusercontent.com/46647141/74587159-31e75180-5015-11ea-86f2-0b840edf04c1.png)

Reply:
> [Check This](https://stackoverflow.com/questions/34395590/emberjs-how-to-import-files-using-root-path)
> 
> ![Screenshot from 2020-02-15 17-03-24](https://user-images.githubusercontent.com/46647141/74587159-31e75180-5015-11ea-86f2-0b840edf04c1.png)

>import Application from '@ember/application';
import Resolver from './resolver';
import loadInitializers from 'ember-load-initializers';
import config from 'open-event-frontend/config/environment';
const App = Application.extend({
  modulePrefix    : config.modulePrefix,
  podModulePrefix : config.podModulePrefix,
  Resolver
});
loadInitializers(App, config.modulePrefix);
export default App;

```

when life is unfair :question: